src/rendering/renderers/gl/WebGLRenderer.ts

```diff
@@ -77,6 +78,8 @@ export class WebGLRenderer extends SystemManager<WebGLRenderer> implements GLRen
         });
 
         this.setup(systemConfig);
+
+        sayHello(this.type);
```

Comment:
inside `StartupSystem` we already have a `hello` property that is false. So you just need to set that to true and replace the console log with `sayHello`

Reply:
ah yes!